src/lib/products.ts

```diff
@@ -108,12 +110,25 @@ export const products: { [product in Product]: ProductData } = {
 		name: "CheerpJ Applet Runner",
 		href: "/cheerpj3/getting-started/Java-applet#running-a-public-applet",
 		logotype: appletrunnerLogotype,
-		favicon: "/cheerpj/favicon.png",
+		favicon: "/cheerpj2/favicon.ico",
 		subtitle: "Run Java Applets in the modern browser",
 		description: "Run Java Applets in modern browsers without plugins",
 		github: "https://github.com/leaningtech/cheerpj-applet-runner",
 		repositoryName: "cheerpj-applet-runner",
 	},
+	"cheerpj-jnlprunner": {
```

Comment:
Pretty clean solution for now. Can you add `// TODO: merge with cheerpj-jnlp-runner` on this